iree/turbine/kernel/wave/expansion/expansion_utils.py

```diff
@@ -87,12 +91,12 @@ def get_dim_scaling(
                 tile_size % wave_count != 0
                 or (tile_size / wave_count) % vector_size != 0
             ):
-                raise ValueError(
```

Comment:
Can we double check that 
```
                tile_size % wave_count != 0
                or (tile_size / wave_count) % vector_size != 0

```
Indeed work for all cases before removing completely, or perhaps need to fine tune this to error out in untested cases